src/client/src/ui/blotter/components/blotterUtils.ts

```diff
@@ -2,11 +2,14 @@ import { ColDef } from 'ag-grid'
 import { Trade, TradeStatus } from 'rt-types'
 import { formatDate, UtcFormatDate } from '../../spotTile/components/notional/utils'
 import SetFilter from './filters/SetFilter'
-
+import numeral from 'numeral'
 const dateRenderer = (trade: Trade, field: string) => {
   return formatDate(trade[field], '%d-%b %H:%M:%S')
 }
 
+const NotionalRenderer = (trade: Trade, field: string) => {
+  return numeral(trade[field]).format('0,000,000[.]00')
```

Comment:
[http://numeraljs.com/](url)

Try `0,0[.]00` instead of `0,000,000[.]00`.

Reply:
Thanks. I have updated it.